pages/index.js

```diff
@@ -28,9 +29,21 @@ const StyledAddSVGContainer = styled.div`
   margin: 1.5rem;
 `;
 
+//----------------------------------------------- FUNCTION------------HERE
+
 let href = "/create";
```

Comment:
I do not think that you need this variable as you could add the value directly to the `Link`: `<Link href="/create"> ...` [(see Next docs for further reference)](https://nextjs.org/docs/pages/api-reference/components/link)

Reply:
I want to reuse my code and I know that in that StyledHeader I am going to put at least another link, that way I can use that variable with a ternary or an if/else.

Reply:
Ok! I really like that you think so much about the reusability of your components. I always prefer to think about how to reuse the component when you need it as until then you often have less not yet needed code. This would make your code much cleaner. That's why I suggested it. :) But feel free to decide for yourself!